lua/pl/compat.lua

```diff
@@ -17,6 +17,8 @@ if isJit then
     compat.jit52 = not loadstring("local goto = 1")
 end
 
+compat.isWindows = package.config:sub(1, 1) == "\\"
+
```

Comment:
@mpeterv what do you think, should path include compat or the otherway around or do we accept a little bit of copy/paste here? Yes, it doesn't need to be public.